components/textUtilities/textUtilities.css

```diff
@@ -0,0 +1,110 @@
+@import url(/components/colors.css);
+/*** headings ***/
+.h1 {
+  font-size: 3rem;
+  line-height: 4rem;
+}
+.h2 {
+  font-size: 2rem;
+  line-height: 2.5rem;
+}
+.h3 {
+  font-size: 1.5rem;
+  line-height: 2rem;
+}
+.h4 {
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+.h5 {
+  font-size: 0.5rem;
+  line-height: 1rem;
+}
+/*******/
+
+/*** text-alignment ***/
+.text-left {
+  text-align: left;
+}
+.text-center {
+  text-align: center;
+}
+.text-right {
+  text-align: right;
+}
+/*******/
+
+/*** text-sizes ***/
+.text-xl {
+  font-size: 2rem;
+}
+.text-lg {
+  font-size: 1.5rem;
+}
+.text-md {
+  font-size: 1rem;
+}
+.text-sm {
+  font-size: 0.875;
+}
+.text-xs {
+  font-size: 0.7rem;
+}
+/*******/
+
+/*** text-transform ***/
+.text-uppercase {
+  text-transform: uppercase;
+}
+.text-lowercase {
+  text-transform: lowercase;
+}
+.text-capitalize {
+  text-transform: capitalize;
+}
+/*******/
+
+/*** font-weights ***/
+.text-light {
+  font-weight: 300;
+}
+.text-regular {
+  font-weight: 400;
+}
+.text-semibold {
+  font-weight: 600;
+}
+.text-bold {
+  font-weight: 700;
+}
+.text-black {
+  font-weight: 900;
+}
+/*******/
+/*** extra-styles ***/
+.text-gray {
+  color: gray;
+}
```

Comment:
Just a suggestion:

text-gray, changes the font color. The user might think that text-black will change the font color to black. Rename name this class, if possible.

Reply:
Ohh I did not realize that. Thanks, I will change it.